addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -692,6 +692,14 @@ blockquote {
                     }
                 }
             }
+            a.list-group-item {
+                color: $-btn-primary-color;
+
+                &.active {
+                    background-color: $-btn-primary-color;
+                    color: color-yiq($-btn-primary-color);
```

Comment:
The bootstrap code seems to do this:
```scss
    color: $list-group-active-color;
    background-color: $list-group-active-bg;
    border-color: $list-group-active-border-color;
```
which means that the border-color might need to be changed too for the style to be correct. Is not there a bug changing the btn-primary-color of a preset?

Reply:
:+1: Done